nutils/evaluable.py

```diff
@@ -1567,8 +1574,7 @@ def _simplified(self):
         if equalindex(self.func.shape[-1], 1):
             return reciprocal(self.func)
 
-    def evalf(self, arr):
-        return numeric.inv(arr)
+    evalf = staticmethod(numeric.inv)
```

Comment:
I haven't checked, but are you sure staticmethod is necessary here? I thought this is implied if the object is not of type function. (same in other similar places)

Reply:
AFAIK a function `func` defined with `def` implements the descriptor protocol such that when calling `func.__get__` with a not-None instance the result is a method call bound to that instance (comparable to `functools.partial(func, instance)`). It doesn't matter if `func` is defined as part of a class or not.

Numpy ufuncs don't implement the descriptor protocol, so wrapping those in a `staticmethod` is unnecessary. I added `staticmethod` for symmetry, but if you don't like it, I'll strip all `staticmethod` from Numpy ufuncs.

Reply:
Right, no I guess there are merits to being explicit, I just wasn't sure if there were reasons beyond that somehow relating to the subsequent commits, especially when I noticed you revisiting the existing Pointwise objects. Let's keep the changes you made.